__tests__/services/fhir.spec.ts

```diff
@@ -396,10 +721,160 @@ describe('Service `fhir`', () => {
 
         await getConcepts(valueSetId, params);
 
-        expect((<jest.Mock>service).mock.calls[0][0]).toEqual({
+        expect(service).toHaveBeenLastCalledWith({
             method: 'GET',
             url: `/ValueSet/${valueSetId}/$expand`,
             params,
         });
     });
+
+    test('method `applyFHIRService`', async () => {
+        const resource = {
+            resourceType: 'Patient',
+        };
+
+        const result = await applyFHIRService(create(resource));
+
+        expect(result).toEqual(success('data'));
+    });
+
+    test('method `applyFHIRServices`', async () => {
+        const result = await applyFHIRServices([
+            create({
+                resourceType: 'Patient',
+            }),
+            update({
+                resourceType: 'Patient',
+                id: '42',
+            }),
+            forceDelete({
+                resourceType: 'Patient',
+                id: '42',
+            }),
+        ]);
+
+        expect(service).toHaveBeenLastCalledWith({
+            url: '/',
+            method: 'POST',
+            data: {
+                type: 'transaction',
+                entry: [
+                    {
+                        request: {
+                            method: 'POST',
+                            url: '/Patient',
+                        },
+                        resource: {
+                            resourceType: 'Patient',
+                        },
+                    },
+                    {
```

Comment:
It is strange that resource is missing. Resources for PUT should be `{
                resourceType: 'Patient',
                id: '42',
            }`

Reply:
See comments below